spec/index.html

```diff
@@ -252,10 +254,10 @@ <h3>Object Lists</h3>
     <p>
       As with predicates often objects are repeated with the same subject and predicate.
```

Comment:
Except that the object does not repeat; the subject and predicate do...
```suggestion
      As with predicates, the same subject and predicate often repeat with different objects.
```